### DISCUSSION TITLE
Как добавить сайт chatgpt.com?

### DISCUSSION THREAD

**Discussion by veleg** [bol-van/zapret#670] [Q&A] [answered]
Не могу разобраться как добавить любые другие сайта для обхода блокировки.

**Comment by oifj34f34f** [comment, marked as answer]
zapret (как и GoodbyeDPI, byeDPI и т.д.) помогает обходить лишь блокировки от РКН, он не меняет ваш IP. Если сайт по своему желанию блокирует пользователей из РФ, то в таком случае поможет лишь VPN, прокси и др технологии для смены IP. То есть в случае c ChatGPT zapret не поможет, т.к. сайт блокирует российские IP

**Comment by ShSeRo** [comment]
А есть предположения? GoodbyeDPI вообще не работает по инструкции, тоесть сайты как не грузились так и не грузятся после запуска cmd

**Reply by HentayHentay** [reply]
одно
Устарело приложение и печальбида

**Comment by dw1rf** [comment]
У меня обход через NextDNS настроен, все сервисы AI работают без впн
